Partition (WBP)/Lists/OL23.lsts.yml

```diff
@@ -0,0 +1,57 @@
+﻿List:
```

Comment:
Remove these extra lists. Put all the objects in LI0036

Reply:
done